lib/fizzy/execute.cpp

```diff
@@ -356,6 +362,49 @@ T fnearest(T value) noexcept
         return t;
 }
 
+float fabs(float input) noexcept
```

Comment:
Could make these templates to avoid the need for casting in `execute`. See `fmax` as an example.

Reply:
Functions should be marked with `inline`.
```suggestion
inline float fabs(float input) noexcept
```